app/controllers/api/v1/catches_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::CatchesController < ApplicationController
```

Comment:
Can add  `protect_from_forgery with: :null_session`

Reply:
this has been added, good catch :)